### PR TITLE
Us2_1_t2 Designed Payment schema, Setting  Up, and testing

### PR DESCRIPTION
**Pull request by FaiiFaiipuipui**
- created a puml Diagram including all current schema and this one.
- created a test file using jest

### CODE REVIEW

**Review by tippaphanun — APPROVED** [review]
I think everything is good.👍🏻

File: models/Transaction.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      required: true,
+    },
+    
+    appointment: {
+      type: mongoose.Schema.ObjectId,
+      ref: "Appointment",
+      required: true,
+    }
+  }
+)
```
Comment (Yiwahpsp) [inline models/Transaction.js:53]: Did you forget toJSON and toObject

**Review by Yiwahpsp — COMMENTED** [review]
Looking Good

File: models/Transaction.js (hunk 1 above)
Comment (FaiiFaiipuipui) [reply models/Transaction.js:54]: Yes I did, thx, please check

File: models/Transaction.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      type: [{
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "transaction_slip"
+      }],
+      default: []
+    },
+
+    successful_payment_slip_image: {
+      type: mongoose.Schema.ObjectId,
+      ref: "transaction_slip",
```
Comment (pccherrii) [inline models/Transaction.js]: This line and line 24, I think their should be 'TransactionSlip'.

Comment (FaiiFaiipuipui) [reply models/Transaction.js]: change done. both seem to work fine on test but will be changed to your recommendation for understandability

**Review by Yiwahpsp — APPROVED** [review]
Good GOoD

**Review by pccherrii — APPROVED** [review]
approve

**Pull request merged**